### ISSUE TITLE
Refresh rate switching kicks to login screen

### ISSUE THREAD

**Issue by Ky-0** [jellyfin/jellyfin-androidtv#1787]
### Describe the bug

Enabling refresh rate switching (any mode) kicks user out back to the login screen when the playback is stopped (exited).

### Application version

0.13.6

### Device information

Nvidia Shield TV (P2987)

### Android version

11

### Jellyfin server version

10.7.7

**Comment by mueslimak3r** [comment]
Are you able to record a log with `adb logcat` so we can see what's going on? Make sure to remove your URL, API key, and other identifying info from it before uploading

**Comment by Ky-0** [comment]
@mueslimak3r
I'd have to have a look at how I do that as I've never touched upon that

**Comment by Ky-0** [comment]
@mueslimak3r 

I have found what the issue was, I changed the folllowing settings on the Andorid TV app and got the following results:

Under Login (Authentication)
Automatic sign in = Disable
System Account = Disable

When I changed Automatic sign in to "Most recently used" it kicked my back to the home screen (not login screen). When I changed both Automatic sign in and System account to "Most recetly used" the issue dissapears - all behaves normally.

So I kind of see why this triggers that bug now.
